### DISCUSSION TITLE
How do I reference the variable agent.ip in the rules-description field?

### DISCUSSION THREAD

**Discussion by yslhzj** [wazuh/wazuh#13845] [Q&A] [answered]
How do I reference the variable agent.ip in the rules-description field?

![图片](https://user-images.githubusercontent.com/29890756/173316402-bca6e8da-1ef3-45ee-8291-a374d3937e43.png)

Thanks

**Comment by HanesSciarrone** [comment, marked as answer]
Hi @yslhzj 

I hope you are well. Thank you for using Wazuh and sharing your question with the community. I'm taking your issue and researching to give you an answer. I'll be back as soon as possible with the answer for you. 

Best regards
Hanes

**Reply by HanesSciarrone** [reply]
Hi @yslhzj 

Unfortunately, there isn't way to use the `agent.ip` variable. The documentation explicitly says that the variables that we can use are the variables parsed by the decoder. The `agent.ip `is a parameter added by Wazuh and the decoder doesn't parse it, it only parses the original log. Wazuh added an option called `location`, in some cases, this parameter in the alert has the agent IP but not always. You can use this parameter of the alert but I'm not sure if you can use it in the description of the rule. I leave you the [description](https://documentation.wazuh.com/current/user-manual/ruleset/ruleset-xml-syntax/rules.html#location) of the location field.

I hope you find the information useful.

Best regards
hanes
